Broken install jrsonnet 0.5.0-pre9

## 詳細
### 詳細症状
examples user で C2A Boom のセットアップのための`npm install` 内で呼ばれる `cargo install --debug --root . jrsonnet --version 0.5.0-pre9` がコケる．

```
   Compiling jrsonnet v0.5.0-pre9
error[E0432]: unresolved import `jrsonnet_evaluator::throw`
  --> /home/sksat/.cargo/registry/src/index.crates.io-6f17d22bba15001f/jrsonnet-0.5.0-pre9/src/main.rs:12:2
   |
12 |     throw, ResultExt, State, Val,
   |     ^^^^^ no `throw` in the root

error: cannot determine resolution for the macro `throw`
   --> /home/sksat/.cargo/registry/src/index.crates.io-6f17d22bba15001f/jrsonnet-0.5.0-pre9/src/main.rs:194:4
    |
194 | ...   throw!("value should be object for --multi manifest...
    |       ^^^^^
    |
    = note: import resolution is stuck, try simplifying macro imports

For more information about this error, try `rustc --explain E0432`.
error: could not compile `jrsonnet` (bin "jrsonnet") due to 2 previous errors
error: failed to compile `jrsonnet v0.5.0-pre9`
```

### 発生条件
C2A の事情には関係なく，`jrsonnet 0.5.0-pre9` 自体が壊れている．
内部でのバージョン付け・参照に不備があり，lock せずにビルドしようとすると内部で `jrsonnet-evaluator v0.5.0-pre95` が使用されてしまう．

###  追加資料
あればリンクなどを貼る

## 必要な作業
- [ ] jrsonnet は lockfile を使ってインストールするようにする

## 影響範囲
example user の C2A Boom

--- a/examples/mobc/tools/install.sh
+++ b/examples/mobc/tools/install.sh
@@ -8,4 +8,4 @@ cargo install --debug --root . tlmcmddb-cli    --version 0.2.0
 cargo install --debug --root . kble            --version 0.2.0
 cargo install --debug --root . kble-c2a        --version 0.2.0
 cargo install --debug --root . kble-eb90       --version 0.2.0
-cargo install --debug --root . jrsonnet        --version 0.5.0-pre9
+cargo install --debug --root . jrsonnet        --version 0.5.0-pre9 --locked

--- a/examples/subobc/tools/install.sh
+++ b/examples/subobc/tools/install.sh
@@ -8,4 +8,4 @@ cargo install --debug --root . tlmcmddb-cli    --version 0.2.0
 cargo install --debug --root . kble            --version 0.2.0
 cargo install --debug --root . kble-c2a        --version 0.2.0
 cargo install --debug --root . kble-eb90       --version 0.2.0
-cargo install --debug --root . jrsonnet        --version 0.5.0-pre9
+cargo install --debug --root . jrsonnet        --version 0.5.0-pre9 --locked

